parsing/entry.mli

```diff
@@ -10,8 +10,9 @@ type should_fail = bool
 
 (** Possible tests in source files. *)
 type test =
-  | Convert of term * term  (** Convertibility between the two given terms. *)
-  | HasType of term * term  (** Typability test, given a term and a type.   *)
+  | Convert  of typed_context * term * term (** Convertibility between the two given terms. *)
+  | HasType  of typed_context * term * term (** Typability test, given a term and a type.   *)
+  | Typeable of typed_context * term        (** Typability test, without the type *)
```

Comment:
Why not call it `infer`?

Reply:
I think it's because it is an assertion, so we are thinking of a predicate rather than a computation. In particular, the Dedukti command `Typeable(c, t)` should not print anything. It could be `IsTypeable`, if you want to stress out the predicate.